dashboard/test/ui/README.md

```diff
@@ -43,23 +43,23 @@ saucelabs_authkey: 'xxxxxx-xxxx-xxxx-xxx-xxxxxxxxx'
 
 You can find the values for these settings in your saucelabs account settings (`https://app.saucelabs.com/users/:username`) The key you need, `saucelabs_authkey`, will be under the "Access Key" header.
 
-#### Saucelabs tunnel
+#### Sauce Labs tunnel
 
-If you want to run tests on saucelabs against localhost you need to set up your tunnel:
+If you want to run tests on Sauce Labs against localhost you need to set up your tunnel:
 
-1. Login to Saucelabs and download the [tunnel](https://app.saucelabs.com/tunnels)
+1. Login to Sauce Labs and download the [tunnel](https://app.saucelabs.com/tunnels)
    - Notes: If you work on a linux EC2 instance, download the linux version (will end in .tar.gz)
      - You will then need to secure copy this file into your dev environment with something like:
      - `scp sc-4.7.1-linux.tar.gz ubuntu@[ip_address]:/ec2-user/environment/code-dot-org`
      - From the destination folder, unzip and untar the file with `tar -xvzf sc-4.7.1-linux.tar`
 2. Start the tunnel via `bin/sc --user <saucelabs-username> --api-key <saucelabs-api-key>`
    - Notes: The link above has an example command line with your credentials that you can copy.
      - `bin/sc` should be replaced with the path that points to the tunnel software downloaded in step 1.
-     - The unique tunnel identifier `-i <tunnel-id>` included in that command can usually be safely ignored (see the note below); if you leave it in, you'll also need to set the `tunnelIdentifier` option in the `sauce_capabilities` config. See [Using Sauce Connect Tunnel Identifiers](https://wiki.saucelabs.com/display/DOCS/Using+Sauce+Connect+Tunnel+Identifiers#UsingSauceConnectTunnelIdentifiers-TheBasicsofUsingTunnelIdentifiers) for more details.
-     - If you are running on an EC2 instance, the -i tunnel identifier referred to above is required to launch the tunnel.
+     - The command given by Sauce Labs includes a `--tunnel-name` flag (formally [`--tunnel-id`](https://docs.saucelabs.com/dev/cli/saucectl/run/#--tunnel-name)). This flag can be removed unless on an EC2 instance (see the note below). If you leave it in, you'll also need to set the `tunnelIdentifier` option in the `sauce_capabilities` config. See [Using Sauce Connect Tunnel Identifiers](https://wiki.saucelabs.com/display/DOCS/Using+Sauce+Connect+Tunnel+Identifiers#UsingSauceConnectTunnelIdentifiers-TheBasicsofUsingTunnelIdentifiers) for more details.
```

Comment:
nit: it might be more clear to offer two distinct, alternative steps here, e.g.
1. if on an EC2 instance then... 
2. if _not_ on an EC2 instance then...

And out of curiosity, what is it about an EC2 instance that requires the name?

Reply:
I have no idea why the EC2 instance requires the name––I carried over those instructions from last time.

Good call re: alternative steps. Just updated to reflect that!